pyproject.toml

```diff
@@ -129,7 +129,7 @@ strict = true
 [tool.pytest.ini_options]
 testpaths = "tests"
 xfail_strict = true
-filterwarnings = ["error"]
+filterwarnings = ["error", "ignore::DeprecationWarning:pydantic_ai"]
```

Comment:
Most of our tests use a deprecated API... Should I change all of them in this PR?